examples/Legend.py

```diff
@@ -32,14 +32,13 @@
 win.addItem(s1)
 
 # # option2: generic method------------------------------------------------
-legend = pg.LegendItem((80,60), offset=(70,20))
+legend = pg.LegendItem((80,60), offset=(70,20), itemStyle=0, labelTextSize='10pt')
```

Comment:
In my opinion, it makes more sense to the user to have these as ``enum``, which has been supported in python for a while.
```suggestion
legend = pg.LegendItem((80,60), offset=(70,20), itemStyle=pg.LegendStyle.Classic, labelTextSize='10pt')
```

Reply:
Different implementation, this is not required anymore